bundles/automation/org.eclipse.smarthome.automation.core/src/main/java/org/eclipse/smarthome/automation/core/internal/RuleEngine.java

```diff
@@ -860,6 +860,9 @@ protected void runNow(String ruleUID, boolean considerConditions) {
 
         try {
             clearContext(rule);
+            if (context!=null && !context.isEmpty()){
```

Comment:
This looks like your formatter does not work. There should be a space in front of and one after the `!=`